docs/helm-deployment-aws_cloud.md

```diff
@@ -328,6 +328,56 @@ export EFS_SERVER=$EFS_FS_ID.efs.$AWS_DEFAULT_REGION.amazonaws.com
 # Publish EFS Volume's DNS name (if not already done)
 export EFS_SERVER=<EFS_ID>.efs.<AWS-REGION>.amazonaws.com
 ```
+### Creating a Docker registry pull secret
+
+Since we need to use protected Docker images from Quay.io, you need access to a secret with credentials to be able to pull those images.
+
+* Log in to Quay.io with your credentials:
+```bash
+docker login quay.io
+```
+
+* Generate a base64 value for your dockercfg using one of the following methods. This will allow Kubernetes to access Quay.io:
+
+```bash
+# Linux
+cat ~/.docker/config.json | base64
+```
+
+```bash
+# Windows
+base64 -w 0 ~/.docker/config.json
+```
+
+You can also generate your pull secret and download it from your Quay.io account. Select **Account Settings** > **Generate Encrypted Password** > **Kubernetes Secret**.
```

Comment:
@killerboot I'll remove this line, in case the options are not accessible from robot/service accounts.